src/modules/rc_update/params.c

```diff
@@ -2251,3 +2251,19 @@ PARAM_DEFINE_FLOAT(RC_FLT_SMP_RATE, 50.0f);
  * @group Radio Calibration
  */
 PARAM_DEFINE_FLOAT(RC_FLT_CUTOFF, 10.0f);
+
+/**
+ * Use multiple RC inputs
+ *
+ * The default value of 0 allow RC command only from the first provider.
+ * Setting this to 1 allows RC commands from any provider to be used.
+ * Warning, in this case, if radio control is lost and RC commands are recived via mavlink (see COM_RC_IN_MODE=2),
+ * RC lost fail safe will not be triggered.
+ *
+ * @group Commander
```

Comment:
I added this parameter to the commander group even if it is used in rc_update because I think it makes more sens. Is it ok or should I move it to another group (for example Radio Calibration)